server/routes/visit.ts

```diff
@@ -52,6 +52,39 @@ export default function routes({
       handlers.map(handler => asyncMiddleware(handler)),
     )
 
+  get('/:reference/clear-notifications', async (req, res) => {
+    const reference = getVisitReference(req)
+
+    return res.render('pages/visit/clearNotifications', {
+      errors: req.flash('errors'),
+      formValues: getFlashFormValues(req),
+      backLinkHref: `/visit/${reference}`,
+    })
+  })
+
+  post(
+    '/:reference/clear-notifications',
+    body('clearNotifications', 'No answer selected').isIn(['yes', 'no']),
+    body('clearReason', 'Enter a reason for not changing the booking')
+      .if(body('clearNotifications').equals('yes'))
+      .trim()
+      .notEmpty(),
```

Comment:
Fine for now but we can add min/max length when we find out what the API supports.